### PR TITLE
update tower catalog item

### PR DESCRIPTION
**Pull request by cbudz**
Hi Dayle,

This PR includes the update to the Create Tower Catalog Item files that's in the Provisioning VMs and Hosts guide. 

Thanks,
Chris

### CODE REVIEW

File: common/create-tower-catalog-item.adoc [hunk 1]
```diff
@@ -13,20 +13,19 @@ First, create a catalog:
 
 Then, create an Ansible Tower service catalog item:
 
-. Navigate to menu:Automation[Ansible Tower > Jobs].
-. Click *Ansible Tower Job Templates* and select an Ansible Tower job template.
-. Click  image:1847.png[Configuration] (*Configuration*), then  image:1862.png[Create Service Dialog from this Job Template] (*Create Service Dialog from this Job Template*).
+. Navigate to menu:Automation[Ansible Tower > Explorer], then click on the *Templates* according menu.
```
Comment (dayleparker) [inline common/create-tower-catalog-item.adoc]: Should "according" be "accordion"? Also, when doing a quick runthrough on our test instance, this was called *Job Templates* -- perhaps this is an older instance though at .199?

**Comment by dayleparker** [comment]
Hi Chris,
The updates look great and read clearly. Just one small issue to resolve, and I think we're all good to merge.
Cheers,
Dayle

**Comment by cbudz** [comment]
Thanks @dayleparker. All fixed.

**Comment by dayleparker** [comment]
Thanks Chris!

**Pull request merged**